code/M3D/m3d.h

```diff
@@ -70,11 +70,14 @@ typedef double M3D_FLOAT;
 #endif
 #if !defined(M3D_SMALLINDEX)
 typedef uint32_t M3D_INDEX;
+#define M3D_UNDEF 0xffffffff
 #define M3D_INDEXMAX 0xfffffffe
 #else
 typedef uint16_t M3D_INDEX;
+#define M3D_UNDEF 0xffff
```

Comment:
Double defined symbol, please just define one!

Reply:
Wrong, they are two different symbols, with different values.

Reply:
Ok, thanks for the update.